src/libraries/Common/src/Interop/Unix/System.Native/Interop.GetControlCharacters.cs

```diff
@@ -8,8 +8,8 @@ internal static partial class Interop
     internal static partial class Sys
     {
         [LibraryImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetControlCharacters")]
-        internal static partial void GetControlCharacters(
-            ControlCharacterNames[] controlCharacterNames, byte[] controlCharacterValues, int controlCharacterLength,
+        internal static unsafe partial void GetControlCharacters(
+            ControlCharacterNames* controlCharacterNames, byte* controlCharacterValues, int controlCharacterLength,
```

Comment:
This is now blittable. Does `LibraryImport` do the "efficient thing" here? That is, can it simply mark `GetControlCharacters` as `extern` + `DllImport` or does it have to implement a stub that forwards to the actual `DllImport`?

Reply:
The source generator will only generate a stub if it's needed; if the signature is completely blittable, it won't generate a stub and will just the partial definition of the method as [DllImport].  That said, this method still has an `out byte` parameter, so this change doesn't impact whether a stub is required.

Reply:
Ah, yeah. I missed the `out` on the next line. But good to know it does the efficient thing if it is blittable.